activesupport/lib/active_support/core_ext/range/inverted.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Range
+  # An inverted range is one where the beginning is greater than the end
```

Comment:
would you mind walking me through how you settled upon this definition of inverted?

Reply:
Thanks for taking a look at this PR. It may be moot since @rafaelfranca closed it, but see my reply to your other comment below. Interested in any other feedback.